docs/wiki/research/eODS.md

```diff
@@ -0,0 +1,277 @@
+# enshrined OPERATOR-DELEGATOR SEPARATION (eODS)
+> [!WARNING]
+> This document covers an active area of research, may be outdated at time of reading and subject to future updates, as the design space around Validator roles unbundling evolves.
+## Roadmap tracker
+
+| Upgrade |    URGE     |       Track       |                    Item                    |         Cross-references         |
+| :-----: | :---------: | :---------------: | :----------------------------------------: | :------------------------------: |
+|  eODS   | the Scourge | Staking economics | Solutions to liquid staking centralization | SSF, IL, ePBS, ET, Light Clients |
+## Relevant context 
+Principal–Agent problem, in which the interests of the Agent are not aligned with the interests of the Principal, is part of any delegation, and even more so present in today's staking scene[^1].
+
+Regarding the staking economy, specific market structures have naturally developed, after the Merge, inherited from the perennial distinction between labor and capital.
+Thus, staking has split naturally in two classes of participants, outside protocol level[^2]:
+
+|    Tier    |                                                        Current natural separation                                                        | Slashing risk |
+| :--------: | :--------------------------------------------------------------------------------------------------------------------------------------: | :-----------: |
+| Delegators |   ETH stakers, with no minimum commitment and no strict requirement to participate in any other way beyond bringing in their principal   |   Slashable   |
+| Operators  | Professional node operators providing sophisticated services, with their reputation or some fixed amount of capital of their own at risk |   Slashable   |
+
+The two tiers are closely interlinked, as liquid staking protocols are not credible if LST holders do not believe that the operators holding their principal are good agents. The risk of slashing, coupled with the high amount of revenue paid out in aggregate to Gasper service providers via issuance, results in intermediated chains of principal-agent relationships[^3] , with Delegators providing capital (Principal) to operators who participate in Gasper on their behalf.
```

Comment:
Appended content with the following:

The risk of slashing, coupled with the high amount of revenue paid out in aggregate to Gasper service providers via issuance, results in intermediated chains of **principal-agent relationships**[^3] , with Delegators providing capital (Principal) to operators who participate in Gasper on their behalf, in order to earn staking rewards for the Principal and staking rewards fees for themselves. 
However, if the validator misbehaves, the funds of the delegator are also slashed, as the actions of the validator (the Agent) affect the capital of the delegator (the Principal).

In order to be considered good agents, honest operators must commit to the Principal's interests:
- perform validation duties properly (i.e. run and maintain good staking infrastructure, be diligent so that its online reputation is upheld by showing high participation rate, and running the prescribed protocol, hence never equivocate by signing conflicting blocks) 
- agents must not be adversarial (malicious), deviating from the protocol arbitrarily, e.g. equivocating and getting slashed in the attempt to obtain under-evaluated stake for the purpose of attacking the network.